redis.conf

```diff
@@ -518,8 +518,9 @@ dir ./
 #    still reply to client requests, possibly with out of date data, or the
 #    data set may just be empty if this is the first synchronization.
 #
-# 2) If replica-serve-stale-data is set to 'no' the replica will reply with
-#    an error "SYNC with master in progress" to all commands except:
+# 2) If replica-serve-stale-data is set to 'no' the replica will reply with error
+#    "MASTERDOWN Link with MASTER is down and replica-serve-stale-data is set to 'no'"
+#    to all commands except:
```

Comment:
The command list here is outdated. Let's change it to suggest that it's a partial list. 
```suggestion
#    to all data access commands, excusing commands such as :
```